mitodl/mit-open
# remove ARTICLE_UI feature flag; bump api version

# What are the relevant tickets?
#134 

# Description (What does it do?)
This removes the article editor feature flag.

# How can this be tested?
1. Ensure `CKEDITOR_ENVIRONMENT_ID, CKEDITOR_SECRET_KEY, CKEDITOR_UPLOAD_URL` are all set in your env file. You can get them from Heroku.
2. Set `MITOPEN_FEATURES_DEFAULT: "False"` in `docker-compose.yml` so that feature flags are not on by default. **As far as I can tell, this has to be done in docker-compose (or overrides), not `.env`**.
3. `docker compose up` and visit http://localhost:8063/api/v1/ckeditor/. You should get a token (and not a 503).
